adapter/googleServiceControl/googleServiceControl.go

```diff
@@ -0,0 +1,116 @@
+package googleServiceControl
+
```

Comment:
I thought that we decide to use all lowercase for filenames such as redisquota

Reply:
Yes, package names should all be lowercase. To that end, maybe googleServiceControl should be shortened to something that looks better in lowercase. I don't have any good suggestions though.

Reply:
shorten to lower case : svcctrl, and will rename the folder to match.